chronos/sendfile.nim

```diff
@@ -38,8 +38,12 @@ when defined(nimdoc):
     ## be prepared to retry the call if there were unsent bytes.
     ##
     ## On error, ``-1`` is returned.
+elif defined(emscripten):
 
-elif defined(linux) or defined(android):
+  proc sendfile*(outfd, infd: int, offset: int, count: var int): int =
+    discard
```

Comment:
While this fixes the compile for me under `emscripten`, it doesn't seem to be actually called by my sample app.

I think that _if_ it ends up being called it would be better to `raiseAssert "Not implemented"` here, or fallback to regular `send`, or return an appropriate error code, rather then return `0` (success).

Reply:
Fixed https://github.com/status-im/nim-chronos/pull/421/commits/6df9b3eb407e5f7101e38de2a2af388ad26d1b3b